packages/ui-utils/src/types.ts

```diff
@@ -117,6 +118,16 @@ export type ToolInvocation =
   | CoreToolCall<string, any>
   | CoreToolResult<string, any, any>;
 
+/**
+ * File object that can be sent to the server.
+ */
+
+export interface MessageFile {
```

Comment:
The name "MessageFile" bugs me a bit but I wasn't able to come up with something better considering that `File` is taken

Reply:
idea: we change the terminology to "Attachment" / "attachments" (then it's clear that it's attached to a message)

Reply:
agreed, updated!